Dockerfile

```diff
@@ -1,6 +1,13 @@
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
 WORKDIR /go/src/github.com/openshift/crd-schema-gen
 COPY . .
+ENV GO_PACKAGE github.com/openshift/crd-schema-gen
 RUN go build -o crd-schema-gen cmd/crd-schema-gen/main.go
-ENV APIS_DIR=vendor/github.com/openshift/api
-ENTRYPOINT ./crd-schema-gen --apis-dir $APIS_DIR
\ No newline at end of file
+
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+WORKDIR /go/src/generator
+COPY --from=builder /go/src/github.com/openshift/crd-schema-gen/crd-schema-gen /
+COPY --from=builder /usr/local/go /usr/local/go
+ENV GOPATH=/go
+ENV APIS_DIR=openshift/api
+ENTRYPOINT /crd-schema-gen --apis-dir $APIS_DIR
```

Comment:
do we really want to hard-code this? I would expect this in the Makefile of consumers.

Reply:
this blocks merging

Reply:
updated this to use `ENTRYPOINT ["/crd-schema-gen"]` and the flag can be passed in with the run command